Bot/BotInstance.cs

```diff
@@ -282,6 +284,8 @@ private async Task<(string secret, string tx, JToken team)> SubmitTeamAsync(stri
                 var cardQuery = CardsCached.Where(x => x.card_detail_id == (string)team["summoner_id"]);
                 string summoner = cardQuery.Any() ? cardQuery.First().card_long_id : null;
                 string monsters = "";
+                var culoare = (string)Settings.CardsDetails[((int)team["summoner_id"]) - 1].GetstringCardColor();
```

Comment:
1. mixing languages in a codebase can be confusing
2. is there an issue this is meant to solve i'm not aware of?

Reply:
I also don't understand this change. 

1.Why is it needed? 
2.Is it ment to be called "color"? It says "culoare"

Reply:
Now, when submitting a team you now need to provide an “allyColor” parameters that matches the element you want to be selected for that summoner, i.e. “Blue”, “Red”, “Green”, “Black”, or “White”., Dragon and Gray is determined by the card colors in the deck.
I changed variable name